src/components/NewsWidget/newsWidget.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { NewsWidget } from '.';
+
+export default {
+  component: NewsWidget,
+  title: 'Pangolin/NewsWidget',
+};
+
+const TemplateNewsWidget: ComponentStory<typeof NewsWidget> = () => <NewsWidget />;
+
+export const Default = TemplateNewsWidget.bind({});
```

Comment:
there are few typescript related issues. if you run `./node_modules/.bin/tsc`
you will see it

<img width="1383" alt="Screenshot 2022-05-18 at 5 54 20 PM" src="https://user-images.githubusercontent.com/6604146/169038093-30972893-5d8e-4f63-b0de-c567a99573eb.png">